pages/windows/dir.md

```diff
@@ -0,0 +1,19 @@
+# DIR 
+
+> Displays file and folders in current directory or given directory.
+
+- Show list of files and folder of current directory:
+
+`dir`
+
+- Show list of file and folder in any other directory:
+
+`dir {{path/to/<placeholder>}}`
```

Comment:
The placeholder is meant to be replaced with some value. Please change it to path/to/directory